users/art/art.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#include QMK_KEYBOARD_H
+#include <string.h>
```

Comment:
```suggestion
```

Reply:
Thank you for your feedback, I've done all the other suggestions but if I try to remove string.h it fails:

users/art/art.c:47:19: error: incompatible implicit declaration of built-in function 'strlen' [-Werror]
     char_to_del = strlen(string1) + strlen(string2);

Reply:
I left a comment about this not working but it seems it didn't work without your other suggestions. Seems fine now, thank you!

Reply:
Sorry, I should have explained this a bit: The string.h include doesn't need to be in the header file, as there's nothing that references anything from it. So, it can be safely moved to art.c instead.